src/Umbraco.Cms.Integrations.Crm.ActiveCampaign/Models/Dtos/FormDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.Integrations.Crm.ActiveCampaign.Models.Dtos
+{
+    public class FormDto
+    {
+        [JsonProperty("id")]
```

Comment:
Do we need both these attributes here?  Seems we should be using `Newtonsoft.Json` or `System.Text.Json` throughout, but not both.